src/GovUk.Education.ExploreEducationStatistics.Admin/Services/EmbedBlockService.cs

```diff
@@ -154,4 +141,29 @@ public class EmbedBlockService : IEmbedBlockService
                     });
             });
     }
+
+    private Either<ActionResult, Unit> ValidateEmbedUrl(string url)
+    {
+        return _options.PermittedEmbedUrlDomains.Split(',').Any(url.StartsWith) 
+            ? Unit.Instance
+            : ValidationActionResult(EmbedBlockUrlDomainNotPermitted);
+    }
+
+    private Either<ActionResult, Unit> ValidateContentSectionAttachedToRelease(
+        Guid releaseId, 
+        Guid contentSectionId)
+    {
+        return _contentDbContext
+            .ReleaseContentSections
+            .Any(rcs => rcs.ReleaseId == releaseId && rcs.ContentSectionId == contentSectionId)
+            ? Unit.Instance
+            : ValidationActionResult(ContentSectionNotAttachedToRelease);
+    }
+    
+    public class ContentOptions
+    {
+        public const string Content = "Content";
+
+        public string PermittedEmbedUrlDomains { get; set; } = null!;
+    }
```

Comment:
Although, we've not done this previously, would suggest we move this into an `Options` namespace so that we know what all the configurable options are for the admin.

Currently it seems like all the options are scattered throughout various classes and it'd be hard to keep track of what options we actually allow!

Reply:
Yeah good idea

Reply:
Removed configuration-per-environment setup, so no longer a need for the ContentOptions here anymore.  I have left an array of permitted options here though, to drive the server-side validation and also left in the ConfigurationController providing this as configuration to the Admin SPA as it'd be nice to only maintain this in one place in Admin. I've shortened the name to `PermittedDomains` too, given the context of the service.